helmfile/charts/grafana-dashboards/dashboards/cluster-api-autoscaling-dashboard.json

```diff
@@ -0,0 +1,1385 @@
+        "w": 4,
+        "x": 12,
+        "y": 1
+      },
+      "id": 12,
+      "links": [],
+      "maxDataPoints": 100,
+      "options": {
+        "colorMode": "background",
+        "graphMode": "none",
+        "justifyMode": "auto",
+        "orientation": "horizontal",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "/^ pods$/",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "9.5.5",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "expr": "sum(cluster_autoscaler_unschedulable_pods_count{pod=~\"^$cluster.*\"})",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": " pods",
+          "range": true,
+          "refId": "A",
+          "step": 300
+        }
+      ],
+      "title": "Number of unscheduled pods",
+      "type": "stat"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${datasource}"
+      },
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "mappings": [
+            {
+              "options": {
+                "match": "null",
+                "result": {
+                  "text": "N/A"
+                }
+              },
+              "type": "special"
+            }
+          ],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "s"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 4,
+        "w": 4,
+        "x": 16,
+        "y": 1
+      },
+      "id": 7,
+      "links": [],
+      "maxDataPoints": 100,
+      "options": {
+        "colorMode": "none",
+        "graphMode": "none",
+        "justifyMode": "auto",
+        "orientation": "horizontal",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "9.5.5",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "expr": "sum(time()-cluster_autoscaler_last_activity{activity=\"scaleDown\", pod=~\"^$cluster.*\"})",
+          "format": "time_series",
+          "intervalFactor": 2,
+          "legendFormat": "",
+          "range": true,
+          "refId": "A",
+          "step": 600
+        }
+      ],
+      "title": "Last scaleDown activity",
```

Comment:
Will this show the last time the autoscaler checked if the cluster should be scaled down (same question for the box showing scale up activity)?
Would it be possible to also show when it last actually scaled up/down?

Reply:
Yes you're correct, these show when the autoscaler tries to or checks scale up/down. There is a different "activity" that might show actual scaling but I'm not sure I understand the description of that metric but I can try it out and add it if that shows actual scaling events. Otherwise you should see it in the node activity time series graph.